browser/ui/views/sidebar/sidebar_item_add_button.h

```diff
@@ -48,6 +48,9 @@ class SidebarItemAddButton : public SidebarButtonView,
   void OnButtonPressed();
 
   raw_ptr<BraveBrowser> browser_ = nullptr;
+  raw_ptr<views::Widget> add_item_bubble_widget_ = nullptr;
+  // true when bubble is shown by add item button press. Otherwise, false.
```

Comment:
Nit/Suggestion:

```suggestion
  // true when bubble is shown by pressing add item button. Otherwise, false which means the bubble is shown by hovering.
```